iaso/api/profiles.py

```diff
@@ -420,16 +420,23 @@ def send_email_invitation(self, profile, email_subject, email_message):
             protocol=protocol,
             domain=domain,
             site_name=site_name,
+            account_name=account.name,
```

Comment:
I think it would be better to retrieve the account name from the profile:

```suggestion
            account_name=profile.account.name,
```

And, therefore, remove the line 409:
```
account = self.request.user.iaso_profile.account
```